branching_xblock/branching_xblock.py

```diff
@@ -12,6 +12,9 @@
 
 resource_loader = ResourceLoader(__name__)
 
+import logging
+logger = logging.getLogger(__name__)
+
 
 class BranchingXBlock(XBlock):
```

Comment:
I think you should add a `display_name` to this block so it has some text to display, even if it can't be changed. If ti can be changed authors will be able to give this problem name.